docs_source/source/documentation.rst

```diff
@@ -89,19 +89,15 @@ To install AxonDeepSeg, you just need to install it with ``pip`` using the follo
 
 
 .. NOTE ::
-   In order to test the installation, you can launch the integrity_test.py script from the terminal. First, go to your virtual environment folder, and then go to the installation folder of AxonDeepSeg (replace NAME_VIRTUALENV by the name of your virtual environment):
+   In order to test the installation, you can launch an integrity test by running the following command on the terminal (make sure your virtual env is activated before, as explained in the previous section):
    ::
-    
-    cd ~/NAME_VIRTUALENV/lib/python2.7/site-packages/AxonDeepSeg
-    python integrity_test.py
+    axondeepseg_test
 
-   If the test succeeds, the following message will appear in the terminal, meaning that the  software was correctly installed:
+   This integrity test automatically performs the axon and myelin segmentation of a test sample. If the test succeeds, the following message will appear in the terminal, meaning that the  software was correctly installed:
```

Comment:
This is a duplicate of the file `docs/_sources/documentation.rst.txt`.
Something is going wrong here. 
@perone can you please help @alzaia fix this? There are obviously issues in the way Github is used here.

Reply:
Sure. This may seem confusing but it is just because we're serving the documentation from the `/docs` repository folder. To make things clear:
1. The documentation sources (Sphinx sources) are located in `/docs_source`;
2. These documentation sources are rendered by Sphinx and then they are commited into the `/docs` repository folder;
3. When we access the site: https://neuropoly.github.io/axondeepseg/, it will serve the rendered documentation that is presented at `/docs` folder.

These files are replicated because Sphinx has an option (that we can disable), where it allows anyone to see the source of the rendered documentation (**see below**):
![image](https://user-images.githubusercontent.com/412328/35347360-ea7ebe86-0102-11e8-92d0-376b9adc29a1.png)

And when this option is enabled, Sphinx copies the source documentation to this folder called `_sources' inside the rendered documentation folder (`docs`). We can disable that, what do you guys think ?

Reply:
I see, that makes sense. If we host the documentation on ReadTheDocs, do we have to manually update the repos in case the doc changes on GH?
If not, then I suggest we do that. I hate seeing duplicated files on a repos (source of future errors in case someone like me doesn't understand how ReadTheDocs works).

Reply:
In any case, we should address this in another PR. Issue opened here: https://github.com/neuropoly/axondeepseg/issues/43

Reply:
Ok, makes sense.